application/frontend/components/BaseRestController.php

```diff
@@ -36,7 +36,7 @@ public function actionIndex()
         $this->throwNotAllowed();
     }
 
-    public function actionUpdate()
+    public function actionUpdate($id)
```

Comment:
these are just generic...non-specific to anything we've done with `employee_id`.  They're just following what would be a typical implementation, i.e., a standard view operation expects an id, not an employeeId (as constructed from the regex in the config).  Too bad Yii didn't take an annotation approach on these things, then the url matching could be built right along with the applicable classes and methods.